src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionBase.cs

```diff
@@ -118,6 +119,13 @@ protected set
             _serviceMessageHandler = serviceMessageHandler;
         }
 
+
+        // test
```

Comment:
>test [](start = 11, length = 4)

no such test only code
consider adding it in TestServiceConnection for better test logic